portal/views/portal.py

```diff
@@ -823,18 +832,12 @@ def config_settings(config_key):
         'MEDIDATA_RAVE_ORG',
         'LOCALIZED_AFFILIATE_ORG'
     )
-    if config_key:
-        key = config_key.upper()
-        if not any(key.startswith(prefix) for prefix in config_prefix_whitelist):
-            abort(400, "Configuration key '{}' not available".format(key))
-        return jsonify({key: current_app.config.get(key)})
-    else:
-        config_settings = {}
```

Comment:
Sorry, should've been more clear- we still need the logic in the `else` section (it returns ALL config key/values), but the `else` itself is unnecessary since we exit before the end of the `if` block